openapi-v1.yaml

```diff
@@ -4705,7 +4769,7 @@ paths:
         type: string
         required: true
     get:
-      description: retrieve a list of timestamps from the array fragment info listing in milliseconds
+      description: retrieve a list of timestamps from the array fragment info listing in milliseconds, paginated
```

Comment:
We should also add pagination query params and change the response schema to return pagination metadata.
Changing the response schema will break compatibility, i'm wondering if this change should be moved to v2.